spec/features/workflow/edit_schedule_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+RSpec.feature "Edit the scheduling of an edition" do
```

Comment:
I wonder if we need this feature? Comparing to something like tags, where we also set an automatic default, we only check that we can edit in general. The main feature spec for this seems to check the value is saved, so I think that covers it :-).

Reply:
Now deleted